### ISSUE TITLE
Allow for Python versions > 3.8

### ISSUE THREAD

**Issue by s3alfisc** [d2cml-ai/csdid#40]
Hi all, 

I think it would be great to allow for Python versions > 3.8 =) 

Additionally, have you consider to use one of the python package managers instead of `setuptools`? I have some 
experience with [poetry ](https://python-poetry.org/) and [pixi ](https://github.com/prefix-dev/pixi) & would be more than happy to help you set this up.

**Comment by markkaj** [comment]
I agree that support for Python $\ge$ 3.9 would be super helpful. Are there any plans to update?

**Comment by anzonyquispe** [comment]
Dear @s3alfisc  and @markkaj 
The package is supported for python 3.13 and there is no restriction in the numpy version. 
Best regards
Anzony Quispe

**Comment by alexanderquispe** [comment]
closing issue since @anzonyquispe already fixed the bug.
